src/components/preview/preview.test.js

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import Preview from "./preview.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import previewComp from "../../../cypress/locators/preview/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const testData1 = [256, 275, 300];
```

Comment:
could we name it somehow to understand what for those numbers are ? like `pixelValues` or something similar?